Makefile

```diff
@@ -1,321 +1,259 @@
-HELM_BASE_IMAGE = quay.io/operator-framework/helm-operator
-OPERATOR_SDK_BASE_IMAGE = quay.io/operator-framework/operator-sdk
-CUSTOM_SCORECARD_TESTS_BASE_IMAGE = quay.io/operator-framework/custom-scorecard-tests
-SCORECARD_TEST_BASE_IMAGE = quay.io/operator-framework/scorecard-test
-SCORECARD_TEST_KUTTL_BASE_IMAGE = quay.io/operator-framework/scorecard-test-kuttl
-
-ANSIBLE_IMAGE ?= $(ANSIBLE_BASE_IMAGE)
-HELM_IMAGE ?= $(HELM_BASE_IMAGE)
-OPERATOR_SDK_IMAGE ?= $(OPERATOR_SDK_BASE_IMAGE)
-CUSTOM_SCORECARD_TESTS_IMAGE ?= $(CUSTOM_SCORECARD_TESTS_BASE_IMAGE)
-SCORECARD_TEST_IMAGE ?= $(SCORECARD_TEST_BASE_IMAGE)
-SCORECARD_TEST_KUTTL_IMAGE ?= $(SCORECARD_TEST_KUTTL_BASE_IMAGE)
-
-ANSIBLE_ARCHES:="amd64" "ppc64le" "arm64" "s390x"
-HELM_ARCHES:="amd64" "ppc64le" "arm64" "s390x"
-OPERATOR_SDK_ARCHES:="amd64" "ppc64le" "arm64" "s390x"
-SCORECARD_TEST_ARCHES:="amd64" "ppc64le" "arm64" "s390x"
-SCORECARD_TEST_KUTTL_ARCHES:="amd64" "ppc64le" "arm64"
-# the custom scorecard test image is a scorecard example only
-CUSTOM_SCORECARD_TESTS_ARCHES:="amd64" "ppc64le" "arm64"
-
-export CGO_ENABLED:=0
-.DEFAULT_GOAL:=help
-
-.PHONY: help
-help: ## Show this help screen
-	@echo 'Usage: make <OPTIONS> ... <TARGETS>'
-	@echo ''
-	@echo 'Available targets are:'
-	@echo ''
-	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z0-9_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
-
-##############################
-# Development                #
-##############################
+export GO111MODULE = on
+export CGO_ENABLED = 0
+export PATH := $(BUILD_DIR):$(PWD)/tools/bin:$(PATH)
 
 ##@ Development
 
-.PHONY: all install
-
-all: format test build/operator-sdk ## Test and Build the Operator SDK
-
-install: ## Install the binaries
-	$(Q)$(GOARGS) go install $(GO_BUILD_ARGS) ./cmd/operator-sdk ./cmd/ansible-operator ./cmd/helm-operator
-
-
-# Code management.
-.PHONY: format tidy clean cli-doc lint
-
-format: ## Format the source code
-	$(Q)go fmt $(PKGS)
-
-tidy: ## Update dependencies
-	$(Q)go mod tidy -v
-
-clean: ## Clean up the build artifacts
-	$(Q)rm -rf build
-
-lint: ## Install and run golangci-lint checks
-ifneq (${GOLANGCI_LINT_VER}, "$(shell ./bin/golangci-lint --version 2>/dev/null | cut -b 27-32)")
-	@echo "golangci-lint missing or not version '${GOLANGCI_LINT_VER}', downloading..."
-	curl -sSfL "https://raw.githubusercontent.com/golangci/golangci-lint/v${GOLANGCI_LINT_VER}/install.sh" | sh -s -- -b ./bin "v${GOLANGCI_LINT_VER}"
-endif
-	./bin/golangci-lint --timeout 5m run
-
-setup-k8s:
-	hack/ci/setup-k8s.sh ${K8S_VERSION}
-
-##############################
-# Generate Artifacts         #
-##############################
-
-##@ Generate
-
-.PHONY: generate cli-doc changelog samples
-
-generate: cli-doc samples bindata ## Run all non-release generate targets
-
-cli-doc: ## Generate CLI documentation
+.PHONY: generate
```

Comment:
why not add the samples to be gen here?

Reply:
I'm not sure I understand the question. `make generate` calls `go run ./hack/generate/samples/generate_all.go` to generate the samples.

Reply:
``` $ make generate
mkdir -p /Users/camilamacedo/go/src/github.com/operator-framework/operator-sdk/build && go build -gcflags "all=-trimpath=/Users/camilamacedo/go/src/github.com/operator-framework" -asmflags "all=-trimpath=/Users/camilamacedo/go/src/github.com/operator-framework" -ldflags " -X 'github.com/operator-framework/operator-sdk/internal/version.Version=v1.1.0+git' -X 'github.com/operator-framework/operator-sdk/internal/version.GitVersion=v1.1.0-7-g382a9980' -X 'github.com/operator-framework/operator-sdk/internal/version.GitCommit=382a998043ddf1959c77063098ac149e90e37ff1' -X 'github.com/operator-framework/operator-sdk/internal/version.KubernetesVersion=v1.18.8' "  -o /Users/camilamacedo/go/src/github.com/operator-framework/operator-sdk/build ./cmd/operator-sdk ./cmd/ansible-operator ./cmd/helm-operator
go run ./hack/generate/cli-doc/gen-cli-doc.go
go run ./hack/generate/samples/generate_all.go
```

Checked. I probably missed it in the GH diff 👍